src/test/groovy/pl/tomaszdziurko/jvm_bloggers/view/admin/moderation/BlogPostItemPopulatorSpec.groovy

```diff
@@ -0,0 +1,41 @@
+package pl.tomaszdziurko.jvm_bloggers.view.admin.moderation
+
+import org.apache.wicket.markup.repeater.Item
+import org.springframework.test.util.ReflectionTestUtils
+import pl.tomaszdziurko.jvm_bloggers.blog_posts.domain.BlogPost
+import pl.tomaszdziurko.jvm_bloggers.utils.NowProvider
+import spock.lang.Specification
+
+import java.time.LocalDateTime
+import java.time.Month
+
+/**
+ * @autor mszarlinski
+ */
+class BlogPostItemPopulatorSpec extends Specification {
+
+    BlogPostItemPopulator objectUnderTest = new BlogPostItemPopulator()
```

Comment:
I think this name is too general, why not just `blogPostItemPopulator`?

Additionally, according to Spock conventions, test subject should be annotated with `@Subject` annotation.